server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/DeleteMailHelper.java

```diff
@@ -47,7 +47,6 @@
         this.deletedMailsDao = deletedMailsDao;
         this.browseStartDao = browseStartDao;
         this.browseHelper = browseHelper;
-
         this.configuration = configuration;
         this.random = new Random();
```

Comment:
first, you should not use Random as there are better alternatives (ThreadLocalRandom or SecureRandom depending of the use case), second, you should take the random number generator in parameter for testability